### PR TITLE
updated the tox and travis configs -- BOM-1557

### PR DESCRIPTION
**Pull request by mraarif**
Relevant JIRA issue [here](https://openedx.atlassian.net/browse/BOM-1557).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -94,15 +94,10 @@ def load_requirements(*requirements_paths):
...
-        'Framework :: Django :: 1.11',
-        'Framework :: Django :: 2.0',
-        'Framework :: Django :: 2.1',
         'Framework :: Django :: 2.2',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)',
         'Natural Language :: English',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
```
Comment (morenol) [inline setup.py:101]: Since the python 3.8 tests are passing, we could add in here the classifiers for python 3.8

Comment (mraarif) [reply setup.py:101]: done.

File: tox.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,35}-django{111}-drf{37,38,39,latest},py{35}-django{20,21,22}-drf{37,38,39,latest}
+envlist = py{35,38}-django{22,30}-drf{37,38,39,latest}
```
Comment (morenol) [inline tox.ini:2]: Django 3.0 dropped support for Python 3.5, so that combination wouldn't work.

File: tox.ini [hunk 3]
```diff
@@ -24,44 +24,41 @@ addopts = --cov completion --cov-report term-missing --cov-report xml
...
-    django111: Django>=1.11,<2.0
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
+deps = 
     django22: Django>=2.2,<2.3
+    django30: Django>=3.0,<3.1
     drf37: djangorestframework<3.8.0
     drf38: djangorestframework<3.9.0
-    drf39: djangorestframework<4.0.0
+    drf39: djangorestframework<3.10.0
```
Comment (mraarif) [inline tox.ini:32]: fixed the dependency to install the correct version, Django rest framework has 3.10 after 3.9 ([source](https://www.django-rest-framework.org/community/release-notes/#versioning))

**Pull request merged**